terraform-dev/main-gcp.tf

```diff
@@ -144,10 +140,6 @@ variable "bigquery_functions_data_viewer_members" {
   type = list(string)
 }
 
-variable "bigquery_graph_data_viewer_members" {
```

Comment:
Yeah this doesn't look like it's doing anything anymore. This will also need removing from `environment.auto.tfvars`.